Rebuild-Black-Business/RBB-Website
# Fix TaC accessibility and set up hover states for Footer links

# Describe your PR
Added hover states to all links in footer and fixed Terms and Conditions color contrast issue.

Fixes color contrast issue for Terms and Conditions and adds hover state to photography credit links. Hex and opacity values now match Figma.

## Pages/Interfaces that will change
All non-social Footer links

### Screenshots / video of changes

Before:
![image](https://user-images.githubusercontent.com/20157895/84103487-c583ca00-a9d8-11ea-986d-d5e244f4e1b4.png)

After:
![image](https://user-images.githubusercontent.com/20157895/84103509-d3d1e600-a9d8-11ea-8aa2-053c21cbc754.png)


## Steps to test

1. Hover over photography credit
2. -> See hover state

1. Run color contrast check
2. -> Check passes
